packages/reflect-core/lib/radial-gradient/radial-gradient.manifest.ts

```diff
@@ -0,0 +1,49 @@
+import type { Color } from "../color";
+import { Alignment } from "../alignment";
+import { GradientType } from "../gradient";
+
+/**
+ * reflect's global linear gradient manifest
+ * [flutter#RadialGradient](https://api.flutter.dev/flutter/painting/RadialGradient-class.html)
+ * [css#radial-gradient](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/radial-gradient())
+ */
+export interface RadialGradientManifest {
+    readonly _type: GradientType.RADIAL;
+    /**
+     * The offset at which center 0.0 of the gradient is placed.
+     */
+    center?: Alignment;
+    /**
+     * The radius that determines the circular shape
+     */
+    radius?: number;
+    /**
+     * The colors the gradient should obtain at each of the stops.
+     */
+    colors: Array<Color>;
+    /**
+     * A list of values from 0.0 to 1.0 that denote fractions along the gradient.
+     */
+    stops?: Array<number>;
+}
+
+export class RadialGradient implements RadialGradientManifest {
+    readonly _type = GradientType.RADIAL;
+
+    readonly center: Alignment;
+    readonly radius: number;
```

Comment:
does not match interface - should be `readonly radius?: number;`

Reply:
fixed. Thank you.